js/main.js

```diff
@@ -1,26 +1,112 @@
-function getRandomNumber(min, max) {
-  let getQuantity = Math.floor(Math.random() * (max - min + 1) + min);
-  if (min >= max) {
-    getQuantity = (Math.random() * (max - min + 1) + min);
-  }
-  return getQuantity;
+function getRandomInteger(min, max) {
+  const lower = Math.min(Math.abs(min), Math.abs(max));
+  const upper =  Math.max(Math.abs(min), Math.abs(max));
+  const randomNumber = (Math.random() * (upper - lower) + lower);
+  return Math.floor(randomNumber);
 }
 
+getRandomInteger(20, 50);
+
 
-for (let i = 1; i <= 10; i++) {
-  getRandomNumber(0, 50);
+function getRandomNumberFloatingPoint(minNumber, maxNumber, floatingPoint) {
+  const lower = Math.min(Math.abs(minNumber), Math.abs(maxNumber));
+  const upper = Math.max(Math.abs(minNumber), Math.abs(maxNumber));
+  const  randomNumberFloat = (Math.random() * (upper - lower) + lower);
+  return randomNumberFloat.toFixed(floatingPoint);
```

Comment:
just FYI: if you need to return a number, you need to use parseFloat before return, as toFixed returns a string